packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -74,6 +74,9 @@ foreman-installer --scenario foreman --migrations-only > /dev/null
 %{_mandir}/man8
 
 %changelog
+* Tue Nov 06 2018 Ondrej Prazak <oprazak@redhat.com> - 1.18.3-1
```

Comment:
This needs to be `1:1.18.3-1` since we now correctly take the epoch into account (as we should have done before).